roxctl/declarativeconfig/lint/lint.go

```diff
@@ -31,31 +32,55 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 
 	cmd.Flags().StringVarP(&lintCmd.file, "file", "f", "", "file containing the declarative configuration in YAML format")
 
-	utils.Must(cmd.MarkFlagRequired("file"))
+	cmd.Flags().String(configmap.ConfigMapFlag, "", `config map from which to read the declarative configuration from.
+In case this is not set, the declarative configuration will be read from the YAML file provided via the --file flag.`)
+	cmd.Flags().String(configmap.NamespaceFlag, "", `namespace of the config map from which to read the declarative configuration from.
+In case this is not set, the namespace set within the current kube config context will be used`)
+
+	cmd.MarkFlagsMutuallyExclusive("file", configmap.ConfigMapFlag)
+
 	return cmd
 }
 
 type lintCmd struct {
 	env environment.Environment
 
 	file         string
-	fileContents []byte
+	fileContents [][]byte
+
+	configMap string
+	namespace string
 }
 
-func (l *lintCmd) Validate() error {
+func (l *lintCmd) Construct(cmd *cobra.Command) error {
+	configMap, namespace, err := configmap.ReadConfigMapFlags(cmd)
+	if err != nil {
+		return errors.Wrap(err, "reading config map flag values")
+	}
+	l.configMap = configMap
+	l.namespace = namespace
+
+	if l.configMap != "" {
+		contents, err := configmap.ReadFromConfigMap(context.Background(), l.configMap, l.namespace)
```

Comment:
Should we put timeout on the context here?

Reply:
I'll have the timeout within `ReadConfigMap`, that makes things a lot easier for clients.